src/binding.cc

```diff
@@ -1071,6 +1071,35 @@ v8__String__GetExternalStringResourceBase(const v8::String& self,
   return self.GetExternalStringResourceBase(encoding_out);
 }
 
+class ExternalOneByteString : public v8::String::ExternalOneByteStringResource {
```

Comment:
i'm curious if we could use binding.hpp here.

Reply:
Gave it a go, ended up being a little annoying (mostly because of [constructors](https://github.com/rust-lang/rust-bindgen/issues/1117) and allowlisting types), but not too bad.